icgcget/commands/versions.py

```diff
@@ -0,0 +1,16 @@
+import logging
```

Comment:
Please add copyright header, and pretty please follow this as mentioned before: https://www.jetbrains.com/help/pycharm/2016.1/c